src/backend/api/backend.did

```diff
@@ -1,3 +1,39 @@
+type Err = record {
+  code : nat16;
+  message : text;
+};
+
+type UserConfig = variant {
+  admin : record {
+    bio : text;
+  };
+  reviewer : record {
+    bio : text;
+    neuron_id : nat64;
+    wallet_address : text;
+  };
+  anonymous;
+};
+
+type GetMyUserProfileResponse = variant {
+  ok : record {
+    id : text;
+    username : text;
+    config : UserConfig;
+  };
+  err : Err;
+};
+
+type CreateMyUserProfileResponse = variant {
+  ok : record {
+    id : text;
+    username : text;
+    config : UserConfig;
+  };
+  err : Err;
+};
+
 service : {
-  "say_hello" : () -> (text) query;
+  get_my_user_profile : () -> (GetMyUserProfileResponse) query;
```

Comment:
The APIs are resource-oriented, sticking with a REST CRUD style as much as possible with an additional convention of using "my" to indicate endpoints that return data specific to the caller. I think this will help the APIs to be predictable and consistent. I take a lot of inspiration from [Google's gRPS API design guide](https://cloud.google.com/apis/design).